src/transformers/models/auto/tokenization_auto.py

```diff
@@ -250,10 +250,9 @@ def from_pretrained(cls, pretrained_model_name_or_path, *inputs, **kwargs):
             pretrained_model_name_or_path (:obj:`str`):
                 Can be either:
 
-                    - A string with the `shortcut name` of a predefined tokenizer to load from cache or download, e.g.,
-                      ``bert-base-uncased``.
-                    - A string with the `identifier name` of a predefined tokenizer that was user-uploaded to our S3,
-                      e.g., ``dbmdz/bert-base-german-cased``.
+                    - A string, the `model id` of a predefined tokenizer hosted inside a model repo on huggingface.co.
+                      Valid model ids can be located at the root-level, like ``bert-base-uncased``, or namespaced under
+                      a user or organization name, like ``dbmdz/bert-base-german-cased``.
```

Comment:
I apologize for this change that's orthogonal to this PR. Will propagate this change everywhere if the wording is agreed upon.

Reply:
This makes more sense so it's good to go for me!